lib/semian/circuit_breaker.rb

```diff
@@ -51,14 +51,13 @@ def mark_failed(error)
 
     def mark_success
       return unless half_open?
-      @success_count += 1
+      push_time(@successes, @success_count_threshold)
```

Comment:
No because of the `return unless half_open?`. So it would happen only while recovering after an outage. I'm not sure if we really need to store time for successes though.